src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,12 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+import React from 'react';
+import { User } from '../../types/User';
+
+type Props = {
+  user: User
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <a className="UserInfo" href="mailto:Sincere@april.biz">
```

Comment:
wait, `mailto` must be dynamic - each user has different email, right?
you can use curly braces here
```suggestion
  <a className="UserInfo" href={`mailto:${user.email}`}>
  //not sure of the outer curly braces though 🤔 prob this would work fine
  // href=`mailto...
```